tests/tools/cohort_tracking/test_cohort_tracking.py

```diff
@@ -0,0 +1,145 @@
+from pathlib import Path
+
+import pytest
+
+import ehrapy as ep
+from ehrapy.io._read import read_csv
+
+CURRENT_DIR = Path(__file__).parent
+_TEST_DATA_PATH = f"{CURRENT_DIR.parent}/test_data_features_ranking"
+_TEST_IMAGE_PATH = f"{CURRENT_DIR.parent}/_images"
+
+
+@pytest.fixture
+def adata_mini():
```

Comment:
Eventually we should move all datasets into the `conftest.py` as fixtures. Also another issue....

Reply:
Opened issue #665 to this 👍